test/90.order.js

```diff
@@ -0,0 +1,164 @@
+/*global describe, it */
```

Comment:
for the tests, it would be nice if it would show an object going in unsorted and coming out sorted. Currently it's requesting for data that has already been saved, but where is the object that was saved? Would be nice to see it in the same test.

Reply:
The objects that were saved are up in the bulk tests.

Reply:
also it looks like the files are already sorted, so would be good to test with something that is unsorted and prove it's coming back sorted.